js/index.js

```diff
@@ -1,5 +1,38 @@
+function loadData() {
+  const data = document.getElementById('data');
+  if (localStorage.getItem('books')) {
+    data.innerHTML = localStorage.getItem('books');
+  }
+}
```

Comment:
- [ ] Great job for working on the loading the data functionality so far but you have a small issue to mention since after refreshing without removing anybooks all the books is removed so please make sure the data is presist with refreshing the page that means when the user refresh not lose the data displayed 🤩🚀💯



https://user-images.githubusercontent.com/58553711/234013750-f34277f2-6f66-4382-8a02-020a6fc1a530.mp4